src/foam/comics/v2/Comics.js

```diff
@@ -200,24 +220,29 @@ foam.CLASS({
       var self = this;
       this.SUPER();
       this
-        .start(this.RowLayout)
-          .start(this.ColumnLayout)
+        .start(this.Rows)
+          .start(this.Cols)
             .add(this.slot(function(data$cannedQueries) {
               return self.E().forEach(data$cannedQueries, function(q) {
```

Comment:
Fixed-ish, I tried to wrap the cols right below .start(this.Cols) and it breaks, so I instead did it right around the .forEach, will probably look at this and fix later but need to get started on the SectionedDetailView since this win't change layout.